js/glotdict-functions.js

```diff
@@ -684,8 +684,8 @@ function gd_build_sticky_header() {
 	toggle_sticky_label.classList.add( 'gd-toggle__label' );
 	toggle_sticky_label.title = 'Stick it and scroll!';
 	toggle_sticky && toggle_sticky.append( toggle_sticky_input, toggle_sticky_label );
-	let glossary_link = document.querySelector( '.gp-heading .glossary-links' );
-	glossary_link && glossary_link.before( toggle_sticky );
+	const first_link = document.querySelector( '.gd-on-translations .gp-content h2 .glossary-link:first-child' );
```

Comment:
Why are you removing the fix that was introduced in a previous PR of yours?
This introduces again the bug where the header toggle is not shown